host-controller/src/main/java/org/jboss/as/host/controller/ServerInventoryImpl.java

```diff
@@ -689,6 +689,20 @@ public void serverProcessStarted(final String serverProcessName) {
     }
 
     @Override
+    public void serverUnstable(final String serverProcessName) {
+        final String serverName = ManagedServer.getServerName(serverProcessName);
+        final ManagedServer server = servers.get(serverName);
+        boolean change = true;
```

Comment:
The variable reflects whether the 'unstable' report represents a 'change' from previous status and hence needs to be reported up the chain.

This is all about doing this part of the reporting once, and then not repeating each time.